src/notifications/getting_started.md

```diff
@@ -0,0 +1,77 @@
+# Implementing mobile notifications
+
+The Breez SDK - *Liquid* Notification Plugin provides developers a simple solution to improve the payment experience on a mobile device. No longer does the application need to be in foreground when receiving payments. When the Notification Plugin is added to process push notifications, the application can be in the background or even closed.
+
+## How it works
+
+The process involves using a Notification Delivery Service (NDS) acting as an intermediary host by the application developer. The NDS must provide a public facing webhook URL where a POST request can be sent to when a notification needs to be delivered to the application. The NDS then forwards the data sent in the webhook POST request via push notification to the application. When the application then receives the push notification, the SDK Notification Plugin can be used to process the event.
+
+![Breez SDK - Liquid Notifications](../images/BreezSDK_Liquid_Notifications.png)
+
+### Push notification data
+
+When receiving POST request data on the webhook URL, the NDS should get the `token`, `platform` and optional `app_data` from the URL params. If necessary, convert the token to a token that can be sent to the push notification service. Then format the data to be sent to the push notification service, which includes converting any JSON data into a string format. 
```

Comment:
For example, if the application uses a non push service token identifier as the token in the webhook URL, then when receiving the POST request if does an internal lookup to map the `token` to the push service `token`

Reply:
Perhaps we can add this to docs for clarification?
```suggestion
When receiving POST request data on the webhook URL, the NDS should get the `token`, `platform` and optional `app_data` from the URL params. If necessary, convert the token into a token compatible with the push notification service. For example, if the application uses a different token identifier in the webhook URL, it will perform an internal lookup to convert this token into a push notification service token upon receiving the POST request. Then format the data to be sent to the push notification service, which includes converting any JSON data into a string format. 
```